src/style.css

```diff
@@ -1,3 +1,73 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  --color-primary: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: "Helvetica", sans-serif;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  margin-left: auto;
+  box-sizing: border-box;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+  list-style: none;
+}
+
+.nav__link {
+  display: flex;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  padding-top: 23px;
+  padding-bottom: 22px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: normal;
+  text-align: center;
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+  &:hover {
+    color: var(--color-primary);
+  }
+}
+
+.is-active {
+  color: var(--color-primary);
+  position: relative;
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  bottom: 0;
+  left: 0;
+  height: 4px;
+  width: 100%;
+  background-color: var(--color-primary);
+  border-radius: 8px;
+}
+
+.container {
+  display: flex;
+  padding-left: 50px;
+  align-items: center;
```

Comment:
In the tests, nav seems to bit lower than it should be. Try to get rid of align-items. Maybe it will help

Reply:
Sadly, tests 3 and 4, but another one puts right side of nav lot higer than it should be ;/